### PR TITLE
mate bumps

### PR DESCRIPTION
**Pull request by oz123**
Some package 1.26.x bumps for mate.

### CODE REVIEW

File: app-arch/engrampa/engrampa-1.26.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (thesamesam) [inline app-arch/engrampa/engrampa-1.26.1.ebuild:4]: Why not meson?

File: dev-libs/libmateweather/libmateweather-1.26.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MATE_LA_PUNT="yes"
+
+inherit mate
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="amd64 ~arm ~arm64 ~loong ~riscv x86"
```
Comment (thesamesam) [inline dev-libs/libmateweather/libmateweather-1.26.1.ebuild:11]: Straight to stable..

**Review by thesamesam — COMMENTED** [review]
I've not commented on the whole thing as it seems like the same issue is there throughout:
- straight-to-stable
- no meson where available

**Comment by oz123** [comment]
Ah, darn. Sorry about these straight to stable.
I'd love to support meson, but not all mate package are ported to meson.
Since all this work is done solely by me ,  I prefer to concentrate on bumping up the packages than to support two build system (e.g. some rebuilds with meson some with Autotools).
I think that mate 1.28 will support meson in all packages. That would be a good time to do the switch.

File: app-arch/engrampa/engrampa-1.26.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+if [[ ${PV} == 9999* ]]; then
+	EGIT_REPO_URI="https://github.com/mate-desktop/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/mate-desktop/${PN}/archive/${P}.tar.xz"
+	KEYWORDS="amd64 ~arm ~arm64 ~loong ~riscv x86"
```
[the remark this answers is not in the document]
Comment (oz123) [reply app-arch/engrampa/engrampa-1.26.1.ebuild:11]: Adding it to my precommit hooks. 🙏

**Pull request closed without merging**